### PR TITLE
Task #16382

### PR DESCRIPTION
**Pull request by SoimanAndrei**

### CODE REVIEW

File: app/assets/front-end/src/javascript/main.controller.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
   function mainController($rootScope, $mdToast, $scope, $state, $timeout) {
 
     var self = this;
```
Comment (victorjeman) [inline app/assets/front-end/src/javascript/main.controller.js]: make use of vm (view model)

File: app/assets/front-end/src/javascript/main.controller.js [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
-
+    $scope.nav = false;
```
Comment (victorjeman) [inline app/assets/front-end/src/javascript/main.controller.js]: use vm.nav

File: app/assets/front-end/src/javascript/main.controller.js [hunk 3]
```diff
@@ -33,6 +33,12 @@
 
         self.isLanding = true;
     };
+    $scope.secNavBar = function(){
```
Comment (victorjeman) [inline app/assets/front-end/src/javascript/main.controller.js]: at the top use:
vm.secNavBar = secNavBar;

and declare that method

function secNavBar() {}

File: app/assets/front-end/src/pages/index.html [hunk 4]
```diff
@@ -29,31 +29,32 @@
         <md-nav-bar
           ng-if="isAdmin"
           class="[ c-navbar c-navbar--admin ] [ js-navbar-admin ]"
+          ng-class="{'secNavBar': nav === true}"
           md-selected-nav-item="currentNavItem"
           nav-bar-aria-label="navigation links"
           ng-hide="mainCtrl.isLanding">
 
-          <md-nav-item md-nav-sref="dashboard" name="Dashboard">
+          <md-nav-item md-nav-sref="dashboard" name="Dashboard" ng-click="firstNavBar()">
```
Comment (victorjeman) [inline app/assets/front-end/src/pages/index.html]: not sure why we are using ng-click, we should use ng-if="isAdmin" or ng-if="!isAdmin" on md-nav-item and just show the links based on that variable

Comment (SoimanAndrei) [inline app/assets/front-end/src/pages/index.html]: I'm using ng-click to change the value of a variable to add a class to md-nav-bar, or to remove it

**Pull request closed without merging**